src/pages/Pair2/index.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { useState } from 'react'
+import { Button, Input } from 'antd'
+import {
+  ShoppingListHeader,
+  GroupCardsContainer,
+  FullScreenContainer,
+  Quantity,
+  ItemName,
+  QuantityContainer,
+  ItemNameContainer,
+  ActionContainer,
+  ItemAdderContainer,
+} from './styles/Pair2.styled'
+import { MinusOutlined, PlusOutlined } from '@ant-design/icons'
+
+type Props = {
+  list: listEntry[]
+  setList: React.Dispatch<React.SetStateAction<listEntry[]>>
+  id: number
+}
+
+type listEntry = {
+  id: number
+  quantity: number
+  itemName: string
+}
+
+type itemAdder = {
+  list: listEntry[]
+  setList: React.Dispatch<React.SetStateAction<listEntry[]>>
+}
+
+const ListHeader = () => {
+  return (
+    <GroupCardsContainer>
+      <QuantityContainer>
+        <h2>Quantity</h2>
+      </QuantityContainer>
+      <ItemNameContainer>
+        <h2>Item</h2>
+      </ItemNameContainer>
+      <ActionContainer>
+        <h2>Actions</h2>
+      </ActionContainer>
+    </GroupCardsContainer>
+  )
+}
+
+const ListItem = (props: Props) => {
+  const list = props.list
+  const setList = props.setList
+  const id = props.id
+
+  const incQuantity = () => {
+    const newList = [...list]
+    newList.forEach((item) => item.id == id && ++item.quantity)
```

Comment:
use 3 = instead of 2

Reply:
fixed in commit 9dba039b3202249cd2782d2f05140593bbd22a95